repo/packages/N/nexus/0/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "3.0",
+  "name": "nexus",
+  "version": "1.0-3.1.0",
+  "minDcosReleaseVersion" : "1.8",
+  "scm": "https://github.com/sonatype/docker-nexus3",
+  "maintainer": "support@sonatype.org",
```

Comment:
It's the Open Source variant. Do you mean the maintainer? The email address was my best guess, as there's no contact address on the website. I'm happy to change this if there's a better suggestion.